fda/fdata_boxplot.py

```
@@ -0,0 +1,627 @@
+        pass
+
+    @abstractmethod
+    def _repr_svg_(self):
+        pass
+
+
+class Boxplot(FDataBoxplot):
+    r"""Representation of the functional boxplot.
+
+    Class implementing the functionl boxplot which is an informative exploratory
+    tool for visualizing functional data, as well as its generalization, the
+    enhanced functional boxplot. Only supports 1 dimensional domain functional data.
+
+    Based on the center outward ordering induced by a :ref:`depth measure <depth-measures>`
+    for functional data, the descriptive statistics of a functional boxplot are: the
+    envelope of the 50% central region, the median curve,and the maximum non-outlying envelope.
+    In addition, outliers can be detected in a functional boxplot by the 1.5 times the 50%
+    central region empirical rule, analogous to the rule for classical boxplots.
+
+    Attributes:
+        fdatagrid (FDataGrid): Object containing the data.
+        median (array, (fdatagrid.ndim_image, nsample_points)): contains
+            the median/s.
+        central_envelope (array, (fdatagrid.ndim_image, 2, nsample_points)):
+            contains the central envelope/s.
+        outlying_envelope (array, (fdatagrid.ndim_image, 2, nsample_points)):
+            contains the outlying envelope/s.
+        colormap (matplotlib.colors.LinearSegmentedColormap): Colormap from
+            which the colors to represent the central regions are selected.
+        central_regions (array, (fdatagrid.ndim_image * ncentral_regions, 2,
+            nsample_points)): contains the central regions.
+        outliers (array, (fdatagrid.ndim_image, fdatagrid.nsamples)):
+            contains the outliers
+        barcol (string): Color of the envelopes and vertical lines.
+        outliercol (string): Color of the ouliers.
+        mediancol (string): Color of the median.
+        fullout (boolean): If true, the entire curve of the outlier samples
```

In the example, by default the curves are shown, and setting this to True hides them. Thus, either the code or the documentation is wrong. I would also change the name for a more understandable one.

---

I have changed it in order the documentation and implementation make sense. Also I changed the name to outliers_repr but I do not like it much, any suggestions?

---

`show_full_outliers`?